docs/data/material/getting-started/templates/sign-in-side/CustomIcons.js

```diff
@@ -0,0 +1,154 @@
+          fill="#4876EE"
+        />
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M153.039 28.6941V32.1013H157.137V21.2585C157.137 19.6234 156.815 18.2898 156.169 17.2569C155.524 16.2028 154.578 15.4176 153.33 14.9013C152.104 14.3851 150.587 14.1268 148.78 14.1268C148.388 14.1268 147.979 14.131 147.553 14.1395C147.301 14.1447 147.043 14.1512 146.779 14.159C146.069 14.1806 145.38 14.2128 144.714 14.2559L144.204 14.2902C143.942 14.3084 143.697 14.3263 143.465 14.3445C143.292 14.358 143.127 14.3715 142.971 14.3851V18.7416C143.638 18.6984 144.38 18.6553 145.198 18.6125C145.426 18.6005 145.653 18.5893 145.878 18.5789C146.203 18.5639 146.523 18.5509 146.841 18.5394C147.091 18.5304 147.339 18.5223 147.586 18.5156C148.36 18.494 148.984 18.4833 149.457 18.4833C150.404 18.4833 151.092 18.7091 151.523 19.161C151.953 19.6128 152.168 20.2904 152.168 21.1941V21.2585H148.973C147.403 21.2585 146.015 21.463 144.81 21.8718C143.606 22.2589 142.67 22.8721 142.003 23.7111C141.336 24.5288 141.003 25.5829 141.003 26.8737C141.003 28.0571 141.271 29.0789 141.809 29.9393C142.347 30.7783 143.089 31.4238 144.036 31.8753C145.004 32.3273 146.123 32.5533 147.392 32.5533C148.618 32.5533 149.651 32.3273 150.49 31.8753C151.351 31.4238 152.017 30.7783 152.491 29.9393C152.574 29.798 152.652 29.6514 152.724 29.499C152.827 29.2836 152.918 29.0565 152.998 28.8193L153.039 28.6941ZM152.168 25.5507V24.1307H148.941C148.037 24.1307 147.338 24.3565 146.844 24.8084C146.37 25.2386 146.133 25.8302 146.133 26.5832C146.133 26.9377 146.192 27.2597 146.31 27.5502C146.429 27.8412 146.607 28.0997 146.844 28.3257C147.338 28.7564 148.037 28.9713 148.941 28.9713C149.113 28.9713 149.279 28.9627 149.44 28.9457C149.571 28.9324 149.698 28.9126 149.821 28.8876C150.058 28.8391 150.281 28.7703 150.49 28.6808C150.964 28.4659 151.351 28.1109 151.651 27.6157C151.953 27.1211 152.125 26.4325 152.168 25.5507Z"
+          fill="#4876EE"
+        />
+        <path
+          d="M160.227 32.1011V14.417H164.326V20.2541C164.522 18.5878 165.017 17.2443 165.811 16.2242C166.886 14.8689 168.435 14.1912 170.458 14.1912H171.135V18.6445H169.844C168.425 18.6445 167.327 19.0316 166.553 19.8062C165.778 20.5592 165.39 21.6564 165.39 23.0977V32.1011H160.227Z"
+          fill="#4876EE"
+        />
+        <path
+          d="M177.682 23.839H179.393L184.976 32.1001H190.784L183.705 21.9545L189.526 14.4161H184.105L177.682 22.8247V8.54287H172.519V32.1001H177.682V23.839Z"
+          fill="#4876EE"
+        />
+      </svg>
+    </SvgIcon>
+  );
+}
+
+SitemarkIcon.propTypes = {
+  sx: PropTypes.oneOfType([
+    PropTypes.arrayOf(
+      PropTypes.oneOfType([PropTypes.func, PropTypes.object, PropTypes.bool]),
+    ),
+    PropTypes.func,
+    PropTypes.object,
+  ]),
+};
+
+export { SitemarkIcon };
```

Comment:
```suggestion

```

Consistency with the others (remove two different ways to export a component in the same file)

Reply:
This is the JS file that's been automatically created by `docs:typescript:formatted`  🤔 
I had to manually change it